src/main/java/org/jabref/gui/ClipBoardManager.java

```diff
@@ -144,6 +144,15 @@ public void setHtmlContent(String html) {
         setPrimaryClipboardContent(content);
     }
 
+    //@overload
```

Comment:
Please remove the comment

Reply:
Can the old `setHtmlContent` method be removed? If it's still used by other code, it might be a good idea to provide a fallback string in these cases as well. What do you think?

Reply:
Hi Tobias,
If I am not mistaken, there is no usage of `setHtmlContent(String html)` anywhere...
Only `setHtmlContent(String html, String fallbackPlain)` is used once on line 230 of `CopyMoreAction.java`.
Should I remove the unused version?

Reply:
Yes, just remove the the unused method.